templates/Japanese fallback/Production/front.html

```diff
@@ -13,12 +13,9 @@
 
 <script>
     function threeDots() {
-        const sent = document.getElementsByClassName("production")[0];
-        if (sent) {
-            for (const hidden of sent.getElementsByTagName("b")) {
-                hidden.style.visibility = "visible";
-                hidden.innerText = "【" + hidden.innerText.replace(/[^ぁ-んァ-ン]/g, "") + "】";
-            }
+        for (hidden of document.querySelectorAll(".production b, .production strong")) {
```

Comment:
You also can add `const` here I think to revert the regression.

Reply:
Well, my comment was a little bit late :^)

Reply:
Probably not a big deal for two commits.